src/components/behandlerdialog/meldinger/MeldingInnholdPanel.tsx

```diff
@@ -88,26 +88,18 @@ const MeldingTekst = ({ melding }: MeldingTekstProps): ReactElement => {
   }
 };
 
-interface AvsenderInfoProps {
+interface MeldingInnholdPanelProps {
   melding: MeldingDTO;
 }
 
-const AvsenderInfo = ({ melding }: AvsenderInfoProps) => {
+export const MeldingInnholdPanel = ({ melding }: MeldingInnholdPanelProps) => {
   const { data: veilederInfo } = useVeilederInfoQuery(
     melding.veilederIdent ?? ""
   );
   const avsender = melding.innkommende
     ? melding.behandlerNavn
     : veilederInfo?.navn;
 
```

Comment:
Er det en ulempe at hele komponenten nå renderes på nytt når `useVeilederInfoQuery` gir et svar? Mest sannsynlig er infoen cachet i reactquery, så vet ikke om det har så mye å si for opplevd hastighet. Men noen ganger kan det jo være en annen veileder som har sendt meldingen, og da er ikke infoen cachet. Vanskelig å teste for, tho 🤷🏼‍♂️

Reply:
Tenkte dette var en case av det John Martin var inne på lenger nede her, at det er litt skjult i koden om komponenten vises eller ikke. Det ser ut som i `MeldingInnholdPanel` at vi får rendret et `AvsenderInfo`, men om vi kikker inn i denne komponenten er ikke dette sikkert likevel 🥲